website_sale_brand_catalogs/controllers/portal.py

```diff
@@ -0,0 +1,32 @@
+from odoo import http
+from odoo.http import request
+
+
+class WebsiteCatalogs(http.Controller):
+    @http.route(
+        [
+            "/catalogs",
+            "/catalogs/page/<int:page>",
+            '/catalogs/brand/<model("product.brand"):brand>',
+            '/catalogs/brand/<model("product.brand"):brand>/page/<int:page>',
+        ],
+        type="http",
+        auth="public",
+        website=True,
+    )
+    def brand_catalogs(self, brand=None, **kw):
+        domain = [("catalogs", "!=", False)]
+        catalog_brand_list = request.env["product.brand"].search(domain)
+
+        values = {}
+        values["catalog_brand_list"] = catalog_brand_list or False
```

Comment:
Instead of creating a dictionary to update it immediately, you should create it will all known values
```suggestion
        values = {"catalog_brand_list": catalog_brand_list or False
```

BTW, why the `or False`? An empty recordset is a falsy value, so I think that's not required